### DISCUSSION TITLE
Warnings on startup

### DISCUSSION THREAD

**Discussion by harshadeva12** [WinmezzZ/react-antd-admin#79] [General]
I'm receiving many warning messages when I run the "npm run start" command. how to fix them. this is a beautiful template and I want to use it. but i cant go further with these warnings. how can I solve them?

![Capture](https://user-images.githubusercontent.com/120442925/222657617-a6b2a590-15b5-42e6-8e12-fb26c7138797.PNG)

**Comment by Who-2u** [comment]
its just a warning from esbuild...add

  esbuild: {
    logOverride: { 'this-is-undefined-in-esm': 'silent' },
  },

in vite.config.ts under defineConfig
